develop/tutorials/articles/01-introduction-to-liferay-development/06-frontend-development-in-liferay/00-frontend-development-in-liferay-intro.markdown

```diff
@@ -17,69 +16,65 @@ technologies you love the most:
 
 ## Loaders [](id=loaders)
 
-To load modules in @product@, you must know when they are needed, where they are 
-located at build time, if you want to bundle them together or load them 
-independently, and you must assemble them at runtime. Keeping track of these 
-tasks can be a hassle. @product@'s Loaders (YUI/AUI and AMD) handle loading for 
-you, so you don't have to worry about all the details. @product@'s loaders are 
-versatile, letting you use YUI/AUI modules, AMD modules, and npm modules 
-(in AMD format). Just provide a small bit of information about your module and 
-@product@'s loaders take care of the rest.
+Ordinarily, loading modules requires you to know when they are needed, where
+they are located at build time, whether you want to bundle them together or load
+them independently, and to assemble them at runtime. Since all that is a hassle,
+@product@'s Loaders (YUI/AUI and AMD) handle it for you. The loaders can use
+YUI/AUI modules, AMD modules, and npm modules (in AMD format). Just provide
+a little information<!--do we have any more detail about this? I hate to leave
+this vague-NR--> about your module and they take care of the rest.
 
 ## npm [](id=npm)
 
-You can use npm in your portlets. To do this, you must create an OSGi 
-bundle with all the npm dependencies extracted and modified to work with the 
-Liferay AMD Loader. Luckily, @product@'s `liferay-npm-bundler` is built for just 
-this purpose, and even provides several presets for common module types 
-(AMD, React, Angular JS,  etc.) to save you time. 
+To use npm in your portlets, create an OSGi bundle with all the npm dependencies
+extracted and modified to work with the Liferay AMD Loader. @product@'s
+`liferay-npm-bundler` is built for this purpose, and speeds the process with
+presets for common module types (AMD, React, Angular JS, etc.). 
 
-Simply include the `liferay-npm-bundler` in your project, and With just a small 
-amount of configuration, the `liferay-npm-bundler` bundles your portlet for OSGi 
-and transpiles your code for the Liferay AMD Loader, which lets you share 
-JavaScript modules and take advantage of semantic versioning among portlets that 
-use the same npm modules on the page.
+Simply include the bundler in your project, and after configuration<!--again,I
+hate to leave this vague. Do we have any information on how exactly the bundler
```

Comment:
See comment above. I keep things as a summary in the intro. The details are covered in the tutorials in this section. Since they don't exist for 7.1 yet, I didn't link to them, but I can add them once they are. Until then, I don't want to go into too much detail in the intro.